chatgpt-on-your-data-2.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Xata & OpenAI: ChatGPT for your data (Part 2)'
+description: 'Use Xata to ask your data questions and get the answers that matter most.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/chatgpt-on-your-data.png
+  alt: Xata and ChatGPT logos
+author: Alexis Rico
+date: 08-29-2023
```

Comment:
Not sure if we want to publish it with launch week?

Reply:
I think it would be great to publish it in the same time with Langchain integrations to make a bigger AI splash.